runtimes/sklearn/mlserver_sklearn/sklearn.py

```diff
@@ -83,7 +83,7 @@ def _get_model_outputs(
         for request_output in payload.outputs:  # type: ignore
             predict_fn = getattr(self._model, request_output.name)
             y = predict_fn(decoded_request)
-            payload = SKLearnPayload(requested_output=request_output, model_output=y)
-            outputs.append(payload)
+            output = SKLearnPayload(requested_output=request_output, model_output=y)
```

Comment:
were these just mypy errors?

Reply:
Yes, exactly. The `payload` variable was already used above with a different type, and `mypy` didn't like that.